### ISSUE TITLE
How to reproduce the result on EPIC-Kitchens dataset

### ISSUE THREAD

**Issue by zyy-student** [swathikirans/GSM#18]
In you paper, you show a great result on EPIC-Kitchens dataset. How to reproduce that. Is there any code for it? Thanks.

**Comment by swathikirans** [comment]
Hi,

EPIC-Kitchens dataset requires a different dataloader and multi-task training due to the structure of the labels and I didn't want to make the repo complicated. The implementation details are available in the paper. You can make the changes to this repo to enable training on EPIC-Kitchens dataset. I will try to integrate the code required for EPIC-Kitchens training to this repo. But I can't confirm now when it will be done. If you have any questions regarding the implementation, please feel free to ask.

**Comment by zyy-student** [comment]
Thanks. I am looking forward to your further works.
